drivers/storage/portworx/preflight.go

```diff
@@ -32,8 +32,10 @@ const (
 	pxPreFlightDaemonSetName          = "px-pre-flight"
 	// PxPreFlightServiceAccountName name of portworx pre flight service account
 	PxPreFlightServiceAccountName = "px-pre-flight"
-	// DefCmetaData default metadata cloud device for DMthin
+	// DefCmetaData default metadata cloud device for DMthin AWS
 	DefCmetaData = "type=gp3,size=64"
+	// DefCmetaVsphere default metadata cloud device for DMthin Vsphere
+	DefCmetaVsphere = "type=eagerzeroedthick,size=64"
```

Comment:
I think we should lazyzeroedthick. Dmthin nodes typically take longer to init. And i suspect eagerzeroedthick might add to that time.

Reply:
lazy zeroed thick apparently has a little performance hit overhead during it's lifetime since there will be zeroes written during IO writes. Do we prefer a little perf overhead later or during initialization?

Reply:
Had an offline discussion and we'll keep it EZT for now until we do some perf testing.